modules/core/AgsCore.swift

```diff
@@ -1,25 +1,44 @@
+import AeroGearHttp
 import Foundation
+import XCGLogger
 
 /**
  * AeroGear Core interface to be used directly by SDK services
  */
 public class AgsCore {
     public static let instance = AgsCore()
 
-    let config: Config
+    let config: ServiceConfig
 
     public init() {
-        config = Config()
-        AgsCoreLogger.logger().debug("Successfully intialized SDK")
+        config = ServiceConfig()
+        AgsCoreLogger.logger().debug("Successfully created AeroGearServices SDK")
     }
 
     /**
      * Get configuration for specific service reference
+     * @param serviceRef unique service reference uset to fetch configuration
+     * @return MobileService instance or nil if configuration is missing service of that type
      */
-    public func getConfiguration(_ serviceRef: String) -> [MobileService] {
+    public func getConfiguration(_ serviceRef: String) -> MobileService? {
         return config[serviceRef]
     }
 
-    public func getLogger() {
+    /**
+     * Get instance of http interface
+     */
+    public func getHttp(_ serviceRef: String) -> Http? {
+        if let service = config[serviceRef] {
```

Comment:
just `return  AgsHttp.getHttp(service: service)` you dont have to unwrap it

Reply:
Cannot get that going. Do you have example?